### PR TITLE
Implemented handling of an Error report of a user

### PR DESCRIPTION
**Pull request by OliveGerste**

### CODE REVIEW

File: src/notification/remote-push-notification/RemotePushNotificationTask.ts [hunk 1]
```diff
@@ -173,6 +173,11 @@ export default class RemotePushNotificationTask implements NotificationTask {
     return Promise.resolve();
   }
 
+  public async sendEndUserErrorNotification(data: EndUserErrorNotification, user: User, tenant: Tenant, severity: NotificationSeverity): Promise<void> {
```
Comment (LucasBrazi06) [inline src/notification/remote-push-notification/RemotePushNotificationTask.ts:176]: I would expect that we also send a remote push notif

Comment (OliveGerste) [reply src/notification/remote-push-notification/RemotePushNotificationTask.ts:176]: Tested in debugger.  
I got the app but the mobileToken for my user will be updated in next db backup for local env. I guess.

Comment (LucasBrazi06) [reply src/notification/remote-push-notification/RemotePushNotificationTask.ts:176]: > Tested in debugger.
> I got the app but the mobileToken for my user will be updated in next db backup for local env. I guess.

Right!
I'll send it to you by email ;-)

Comment (OliveGerste) [reply src/notification/remote-push-notification/RemotePushNotificationTask.ts:176]: Sounds good :)

**Pull request merged**